src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1615,7 +1615,8 @@ inline void Graph::ExecuteNode(const NodePtr& node, SyncInferRequest* request, i
 }
 
 inline void Graph::ExecuteNodeWithCatch(const NodePtr& node, SyncInferRequest* request, int numaId) const {
-    VERBOSE_PERF_DUMP_ITT_DEBUG_LOG(itt::domains::ov_intel_cpu, node, getConfig());
+    // DO NOT MODIFY OR DELETE!! - Domain name cannot change
```

Comment:
Same comment as here: https://github.com/openvinotoolkit/openvino/pull/33311#discussion_r2643387067

Need to introduce better strategy on keeping this not modified